LFS171x/docs/introduction-to-hyperledger.md

```diff
@@ -1 +1,386 @@
+
+\- John Palfreyman, [ibm.com](https://www.ibm.com/blogs/insights-on-business/government/open-innovation-blockchain-hyperledger/)
+
+## The Importance of Open Source (Brian Behlendorf)
+[![The Importance of Open Source (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/BX1o-N9ggHI)
+
+## Open Source and Open Governance
+
+>"Today, most people understand the concept of **Open Source**. What many people don't get, and something we here at Hyperledger and The Linux Foundation pride ourselves on doing well, is **Open Governance**."
+
+\- [hyperledger.org](https://hyperledger.org/blog/2017/09/06/abcs-of-open-governance)
+
+Open source software is software that is made freely available and may be redistributed and modified. In other words, anyone has the ability to view the code, use the code, copy the code, change the code, and, depending on the open source license, contribute back changes.
+
+Open governance means that technical decisions for an open source project are made by a group of community-elected developers drawn from a pool of active participants. These decisions include things such as which features to add, how, and when to add them.
+
+Learn more about the specifics of Hyperledger's open governance at https://hyperledger.org/blog/2017/09/06/abcs-of-open-governance.
+
+## Software Governance of the Hyperledger Projects (Brian Behlendorf)
+[![Software Governance of the Hyperledger Projects (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/8wEPxJS5csc)
+
+## Blockchain for Business
+The cryptocurrency-based blockchain model, popularized by public blockchains like Bitcoin and Ethereum, currently falls short of fulfilling a host of requirements that many types of organizations would have to fulfill in order to be compliant when using blockchain and distributed ledger technologies - for instance, in the areas of financial services, healthcare, and government.
+
+Hyperledger is a unique platform that is developing permissioned distributed ledger frameworks specifically designed for enterprises, including those in industries with strong compliance requirements. Enterprise use cases require capabilities such as scalability and throughput, built-in or interoperable identity modules for the parties involved in a transaction or a network, or even access to regulators who can access all data in the ledger as read-only to ensure compliance. The latter is particularly important because, regardless of the innovation, it has to operate within the current regulatory framework, as well as comply with any new rules that come into place specifically targeted at blockchain technologies.
+
+The enterprise continues to be at the heart of this course.
+
+## Why Businesses Choose to Use Hyperledger? (Brian Behlendorf)
+[![Why Businesses Choose to Use Hyperledger? (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/KX54mejdaOo)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
```

Comment:
Remove - the course is actually licensed under https://creativecommons.org/licenses/by/4.0/. Unfortunately, EdX doesn't let you specify which Creative Common license for videos. Therefore, we will let the overall license cover these.